api/specs.proto

```diff
@@ -386,6 +386,9 @@ message SecretSpec {
 
 	// Data is the secret payload - the maximum size is 500KB (that is, 500*1024 bytes)
 	bytes data = 2;
+
+	// Driver is the name of the secret driver that is used to store the specified secret
+	string driver = 3;
```

Comment:
I wonder if it makes sense to use the `Driver` type here, to allow the future possibility of passing secret-specific options to the driver.